pipeline_steps/common.groovy

```diff
@@ -21,6 +21,9 @@ def install_ansible(){
     cp -r /usr/lib64/python2.6/site-packages/selinux .venv/lib64/python2.7/site-packages/ ||:
     source .venv/bin/activate
 
+    # UG-613 change TMPDIR to directory with more space
+    export PREV_TMPDIR=\$TMPDIR
```

Comment:
It's not from what I've seen either. The PREV_TMPDIR was just for an abundance of caution and to limit TMPDIR changes to just the pip installs, as we don't need to direct all tmp files to this tmp directory